integration/fabric/atsa/chaincode/chaincode/asset_transfer.go

```diff
@@ -437,24 +436,24 @@ func transferAssetState(ctx contractapi.TransactionContextInterface, asset *Asse
 		return fmt.Errorf("failed to create composite key for receipt: %v", err)
 	}
 
-	txTimestamp, err := ctx.GetStub().GetTxTimestamp()
-	if err != nil {
-		return fmt.Errorf("failed to create timestamp for receipt: %v", err)
-	}
-
-	if err := txTimestamp.CheckValid(); err != nil {
-		return err
-	}
-	timestamp := txTimestamp.AsTime()
-
-	assetReceipt := receipt{
-		price:     price,
-		timestamp: timestamp,
-	}
-	receipt, err := json.Marshal(assetReceipt)
-	if err != nil {
-		return fmt.Errorf("failed to marshal receipt: %v", err)
-	}
+	//txTimestamp, err := ctx.GetStub().GetTxTimestamp()
+	//if err != nil {
+	//	return fmt.Errorf("failed to create timestamp for receipt: %v", err)
+	//}
+	//
+	//if err = :txTimestamp.CheckValid(); err != nil {
+	//	return err
+	//}
+	//timestamp := txTimestamp.AsTime()
+	//
+	//assetReceipt := receipt{
+	//	price:     price,
+	//	timestamp: timestamp,
+	//}
+	receipt := []byte{}
```

Comment:
Check under fabric-samples if it is used.

Reply:
https://github.com/hyperledger/fabric-samples/blob/main/asset-transfer-basic/chaincode-go/chaincode/smartcontract.go